Expensify/App
# Make tapping the entire IOU preview component open the IOU Details modal

_NOTE TO REVIEWER: it's better to review it [without whitespace changes](https://github.com/Expensify/Expensify.cash/pull/3541/files?diff=split&w=1)_

### Details
See the issue for more context.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/3539

### Tests
1. Create an IOU request.
2. Tap on the preview.
3. Verify it opens the IOU details modal.

### QA Steps
Same as the above tests

### Tested On


### Screenshots

#### Web

https://user-images.githubusercontent.com/31285285/121630770-5be94080-cab0-11eb-87ef-e758caf3f9e0.mp4


https://user-images.githubusercontent.com/31285285/121630774-5f7cc780-cab0-11eb-998e-3367efa62403.mp4



https://user-images.githubusercontent.com/31285285/121630827-74595b00-cab0-11eb-9fbf-8fd0e6428c2f.mp4


https://user-images.githubusercontent.com/31285285/121630783-61468b00-cab0-11eb-87a0-447967d96062.mp4


https://user-images.githubusercontent.com/31285285/121630784-61df2180-cab0-11eb-994c-82e769dfa315.mp4
